Dockerfile

```diff
@@ -1,39 +1,118 @@
-# Build the manager binary
-FROM golang:1.17 as builder
+FROM --platform=$BUILDPLATFORM crazymax/goreleaser-xx:1.2.5 AS goreleaser-xx
+FROM --platform=$BUILDPLATFORM pratikimprowise/upx:3.96 AS upx
+FROM --platform=$BUILDPLATFORM crazymax/goxx:1.17 AS base
```

Comment:
goxx is not needed here as CGO doesn't seem required in this project. `golang:1.17` is enough.